cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -1069,3 +1252,42 @@ func getDeviceRXBytes(pod *corev1.Pod, device string) (int, error) {
 	}
 	return -1, fmt.Errorf("could not find RX stats: %v", statsLines)
 }
+
+func isContainerUseDevicesSEBooleanDisabled(node string) bool {
+	mcd := getMachineConfigDaemon(node)
+
+	output, err := pods.ExecCommand(client.Client, *mcd, []string{"sh", "-c", "nsenter --mount=/proc/1/ns/mnt -- sh -c 'getsebool container_use_devices'"})
+	Expect(err).ToNot(HaveOccurred())
+
+	return strings.Contains(output.String(), "off")
+}
+func setContainerUseDevicesSEBoolean(node string) {
+	mcd := getMachineConfigDaemon(node)
+
+	_, err := pods.ExecCommand(client.Client, *mcd, []string{"sh", "-c", "nsenter --mount=/proc/1/ns/mnt -- sh -c 'setsebool container_use_devices 1'"})
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func UnsetContainerUseDevicesSEBoolean(node string) {
```

Comment:
Please, consider making it package private

Reply:
thanks

Reply:
done